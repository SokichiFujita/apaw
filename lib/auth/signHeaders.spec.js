const signHeaders = require('./signHeaders');

describe('signHeaders', () => {
  it('should create a formatted string of header keys', () => {
    const headers = {
      'Content-Encoding': 'amz-1.0',
      'X-Amz-Date': '20200308T123600Z',
    };
    const signedHeaders = signHeaders(headers);
    expect(signedHeaders).toMatchSnapshot();
  });
});
