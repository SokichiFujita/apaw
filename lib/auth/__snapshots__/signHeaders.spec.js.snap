// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`signHeaders should create a formatted string of header keys 1`] = `"content-encoding;x-amz-date"`;
